### PR TITLE
add_reset_test.go

### PR DESCRIPTION
**Pull request by Rudolf1998**
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: pkg/yurtadm/cmd/reset/reset_test.go [hunk 1]
```diff
@@ -0,0 +1,327 @@
+/*
+Copyright 2021 The OpenYurt Authors.
```
Comment (Congrool) [inline pkg/yurtadm/cmd/reset/reset_test.go:2]: Please replace the year as 2022.

Comment (Rudolf1998) [reply pkg/yurtadm/cmd/reset/reset_test.go]: Thanks for your suggestions，the bug is already fixed

**Comment by Congrool** [comment]
Thanks!
/lgtm

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**